src/controllers/fortupdate.js

```diff
@@ -0,0 +1,232 @@
+
+			data.name = data.new?.name || data.old?.name || 'unknown'
+			data.name = this.escapeJsonString(data.name)
+			data.description = data.new?.description || data.old?.description || 'unknown'
+			data.imgUrl = data.new?.image_url || data.old?.image_url || ''
+
+			if (data.old) {
+				data.old.imgUrl = data.old.image_url
+			}
+			if (data.new) {
+				data.new.imgUrl = data.new.image_url
+			}
+
+			const whoCares = data.poracleTest ? [{
+				...data.poracleTest,
+				clean: false,
+				ping: '',
+			}] : await this.fortUpdateWhoCares(data)
+
+			if (whoCares.length) {
+				this.log.info(`${logReference}: Fort Update ${data.fortType} ${data.id} ${data.name} found in areas (${data.matched}) and ${whoCares.length} humans cared.`)
+			} else {
+				this.log.verbose(`${logReference}: Fort Update ${data.fortType} ${data.id} ${data.name} found in areas (${data.matched}) and ${whoCares.length} humans cared.`)
+			}
+
+			let discordCacheBad = true // assume the worst
+			whoCares.forEach((cares) => {
+				if (!this.isRateLimited(cares.id)) discordCacheBad = false
+			})
+
+			if (discordCacheBad) {
+				whoCares.forEach((cares) => {
+					this.log.verbose(`${logReference}: Not creating nest alert (Rate limit) for ${cares.type} ${cares.id} ${cares.name} ${cares.language} ${cares.template}`)
+				})
+
+				return []
+			}
+
+			data.shinyPossible = this.shinyPossible.isShinyPossible(data.pokemonId, data.formId)
+
+			data.stickerUrl = data.imgUrl
+
+			const geoResult = await this.getAddress({ lat: data.latitude, lon: data.longitude })
+			const jobs = []
+
+			await this.getStaticMapUrl(logReference, data, 'fort-update', ['map_latitude', 'map_longitude', 'zoom', 'imgUrl', 'poly_path'])
+			data.staticmap = data.staticMap // deprecated
+
+			for (const cares of whoCares) {
+				this.log.debug(`${logReference}: Creating fort update alert for ${cares.id} ${cares.name} ${cares.type} ${cares.language} ${cares.template}`, cares)
+
+				const rateLimitTtr = this.getRateLimitTimeToRelease(cares.id)
+				if (rateLimitTtr) {
+					this.log.verbose(`${logReference}: Not creating fort update (Rate limit) for ${cares.type} ${cares.id} ${cares.name} Time to release: ${rateLimitTtr}`)
+					// eslint-disable-next-line no-continue
+					continue
+				}
+				this.log.verbose(`${logReference}: Creating fort update alert for ${cares.type} ${cares.id} ${cares.name} ${cares.language} ${cares.template}`)
+
+				const language = cares.language || this.config.general.locale
+				const translator = this.translatorFactory.Translator(language)
+				let [platform] = cares.type.split(':')
+				if (platform === 'webhook') platform = 'discord'
+
+				// full build
+				data.name = translator.translate(data.nameEng)
+				data.formName = translator.translate(data.formNameEng)
+				data.shinyPossibleEmoji = data.shinyPossible ? translator.translate(this.emojiLookup.lookup('shiny', platform)) : ''
```

Comment:
these have to go, specifically line 188 overwrites the fort name

Reply:
Thanks, have quickly removed (they are obviously cut and paste nonsense!)